### PR TITLE
Registration form fix

### LINKED ISSUE #55: Registration screen lock form on request
image

during request pending **form must be locked**:
- inputs are disabled
- button is disabled
- at least header loader in enabled (to verify in login has entire **screen spinner**, if so - to be added here to, otherwise it's up to developer's decision to add it or not)

### PR DESCRIPTION
**Pull request by ayarychevska**
#55

### CODE REVIEW

File: src/features/auth/Register/index.tsx [hunk 1]
```diff
@@ -82,13 +74,15 @@ const Register = () => {
...
-    register(form).then((response: any) => {
-      const state = { isRegistered: !response.error };
-      const path = !response.error && ROUTES.AUTH.ROOT;
-      path ? navigate(path, { state }) : setIsEmailError(true);
-    });
+    register(form);
   };
 
+  useEffect(() => {
+    if (registerData && isSuccess) {
```
Comment (doskachok) [inline src/features/auth/Register/index.tsx]: I guess registerData is a bit redundant here. Probably "isSuccess" should be enough

**Review by doskachok — COMMENTED** [review]
Could you please rebase on top of dev, there were some changes in users api/types?
Also it seems like notification about a successful registration is no longer working.

**Comment by doskachok** [comment]
Build: https://magellan2022-613c2--preview-vo6ly14l.web.app

**Review by Gamgeee — APPROVED** [review]

**Pull request merged**